src/Gui/MainWindow.cpp

```diff
@@ -1705,7 +1705,7 @@ void MainWindow::loadWindowSettings()
     max ? showMaximized() : show();
 
     // make menus and tooltips usable in fullscreen under Windows, see issue #7563
-#if defined(Q_OS_WIN)
+#if defined(Q_OS_WIN) && QT_VERSION < QT_VERSION_CHECK(6,0,0)
```

Comment:
Qt6 way:
```
#include <qpa/qplatformwindow_p.h>
    using namespace QNativeInterface::Private;
    if (auto *windowsWindow = dynamic_cast<QWindowsWindow*>(this->windowHandle()))
        windowsWindow->setHasBorderInFullScreen(true);
```
and
`target_link_libraries(FreeCADGui ${FreeCADGui_LIBS} Qt6::GuiPrivate)`

Reply:
Thanks, changed.